models/fivetran_log__audit_table.sql

```diff
@@ -106,8 +134,18 @@ sum_records_modified as (
         and records_modified_log.created_at > limit_to_table_starts.sync_start 
         and records_modified_log.created_at < coalesce(limit_to_table_starts.sync_end, limit_to_table_starts.next_sync_start) 
 
-    group by 1,2,3,4,5,6,7,8,9
+    {{ dbt_utils.group_by(n=9) }}
+
+),
+
+surrogate_key as (
+
+    select 
+        *,
+        {{ dbt_utils.surrogate_key(['connector_id', 'destination_id', 'table_name', 'write_to_table_start']) }} as unique_table_sync_key
```

Comment:
Can you add the new `unique_table_sync_key` to the fivetran_log.yml for this table and regenerate the docs.

Reply:
Also, maybe we can remove the unique combination of columns test and instead simplify the test logic by just applying a `unique` and `not_null` test on the new field.